prboom2/src/MUSIC/portmidiplayer.c

```diff
@@ -610,7 +610,10 @@ static void pm_render (void *vdest, unsigned bufflen)
               eventpos = 0;
               // prevent hanging notes (doom2.wad MAP14, MAP22)
               for (int i = 0; i < 16; i++)
+              {
                 writeevent (when, 0xB0, i, 0x7B, 0x00); // all notes off
+                writeevent (when, 0xB0, i, 0x40, 0x00); // hold1 (damper/sustain) pedal
```

Comment:
Should we just reset device on song loop? It seems `i_winmusic.c` is also affected.

Reply:
The MIDI assoc. only provides a "recommended practice" (RP-015) for how "Reset All Controllers" is supposed to behave so I don't want to risk breaking seamless looping if a device chooses to immediately silences all notes, or takes a long time to execute the event, etc.